src/api/app/models/cloud/upload_job.rb

```diff
@@ -0,0 +1,40 @@
+module Cloud
+  class UploadJob
+    include ActiveModel::Validations
+    include ActiveModel::Model
+    extend Forwardable
+
+    attr_accessor :user, :params
+    attr_reader :user_upload_job, :backend_upload_job
+    validate :validate_jobs
+    def_delegator :backend_upload_job, :id
+
+    def create
+      self.backend_upload_job = Cloud::Backend::UploadJob.create(user, params)
```

Comment:
We usually skip the self. So I would do that here as well.

Reply:
For assignment it is necessary, otherwise it would store it in a local variable ...

Reply:
Ah, you are right